src/audio/buffer.c

```diff
@@ -251,6 +258,7 @@ void comp_update_buffer_produce(struct comp_buffer __sparse_cache *buffer, uint3
 		 (char *)audio_stream_get_addr(&buffer->stream)) << 16 |
 		((char *)audio_stream_get_wptr(&buffer->stream) -
 		 (char *)audio_stream_get_addr(&buffer->stream)));
+*/
```

Comment:
Should this be commented out ?

Reply:
ok, so it would seem the best way is to KCONFIG_SOF_LOG_DBG_BUFFER with a boolean that just #ifdefs these logs. This would allow a Kconfig to enable if anyone needed to debug a buffer.